Client/src/Components/Profile/ProfileGraphList.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Grid } from "@material-ui/core"
+
+import { IGraphStateModel } from '../../Models/Graph/IGraphStateModel'
+import { ProfileGraphRow } from '../Profile/ProfileGraphRow'
+import React from 'react'
+import { toGraphDatasetState } from "../../Models/Graph/IGraphDatasetModel"
+
+interface IGDProps {
```

Comment:
What is IGDProps? Can you rename this so its more clear on what this Iprops is for?